lightning/src/onion_message/messenger.rs

```diff
@@ -921,6 +921,17 @@ where
 		}
 		msgs
 	}
+
+	fn enqueue_event(&self, event: Event) {
+		const MAX_EVENTS_BUFFER_SIZE: usize = (1 << 10) * 256;
```

Comment:
Would like to get some thoughts on this event buffer size (currently 0.34MB). I can also add logging if an event is dropped, though I believe that should be rare.

Reply:
Wouldn't that exclude the majority of the bytes in the onion packet, which are on the heap? Are we trying to limit the number of events or the total size of those events?

Reply:
@valentinewallace Not sure if this was missed.

Reply:
Sorry about that, I thought we were trying to limit the total size of the events. I'm not sure I'm understanding @TheBlueMatt's suggestion :(